lessons/en/chapter_9.yaml

```
@@ -13,12 +34,115 @@
     A module can hold global variables, functions, structs, traits or even other modules!
 
     In Rust there is not a 1 to 1 mapping of files to the module tree hierarchy. We must build the module tree explicitly by hand in our code.
+    
+    
+    More information about how to split you code in different files / moduels
+    can be found [here](https://web.mit.edu/rust-lang_v1.25/arch/amd64_ubuntu1404/share/doc/rust/html/book/first-edition/crates-and-modules.html).
 - title: Writing a Program
   content_markdown: |
     A program has a root module in a file called `main.rs`.
+- title: Binary Crate
+  content_markdown: |
+    For a `binary crate`, `cargo` has the following terminal commands:
+    
+    ```
+    $ cargo new my-first-binary-crates	# creates a binary create
+    $ cd my-first-binary-crates/
+    $ cargo build				# equivalent of `rustc main.rs`
+    $ cargo run				# equivalent of `./main`
+    ```
+
+    If you inspect the file hierarchy system, you'll notice that `cargo`
+    automatically creates some files and folders, and also writes some code.
+    Cool!
+
+    A `binary crate` will depend on the `main.rs`.
+    Btw, do not try to create a `lib.rs` here.
```

I would avoid using abbreviations:
```suggestion
    By the way, do not try to create a `lib.rs` here.
```